docker/registry/cpp-client-base/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=cpp-client-base:1
+deephaven.registry.imageId=cpp-client-base@sha256:7d99ae35e5f732ee1e6df65fe82b3dc6bcb2f38529f67081bd7328caadd203ba
```

Comment:
These are not properly reference imageName / imageId. Should start w/ `ghcr.io/deephaven/...`

Reply:
That explains why they don't work... but it doesn't explain why the directory I used as a template has the same problem and that does work:

https://github.com/deephaven/deephaven-core/blob/4d721823f7d8841c1ff86a7c0ce7d0a220155559/docker/registry/go/gradle.properties